### ISSUE TITLE
Chromecast blocked

### ISSUE THREAD

**Issue by deroverda** [safing/portmaster#384]
Hi everryone. 

I don't know if it's simplewall or portmaster that is blocking access to chromecast. I'm almost certain I've allowed everything that has to do with chrome within Simplewall but when it comes to Portmaster it's a bit more diffuclt since portmaster is showing multiplies connections. I don't know which connection has to do with chromecast. Can anyone help? 

Cheers.

**Comment by PeterKW** [comment]
Yeah, I'm having the same thing here in chromium based browsers. I haven't been able to easily spot the connection request that's being blocked in the list. If anyone is able to get it working that would be great

**Comment by dhaavi** [comment]
Hey @deroverda and @PeterKW, thanks for reporting this! 

We are working on ways to easier find blocked connections and fix apps.

In order to help you, please reproduce the problem and then go to the Network Monitor in the Portmaster UI. There, select the browser you used and press the "Copy Debug Information" button and paste the data here (or on https://bin.privacytools.io/ and supply the link). Please also check the data for potentially sensitive data before posting, especially logs and connections.

What might be happening is that the browser is using mDNS to find the Chromecast. As the Portmaster cannot yet correctly match multicast responses, these would be blocked. If this is the case, you can try fixing this by adding a new allowed [Incoming Rule](https://docs.safing.io/portmaster/settings#filter/serviceEndpoints) to the browser settings with the value `LAN UDP/5353`.

**Comment by PeterKW** [comment]
I believe I may have managed to get it working @dhaavi & @deroverda 

I collected the IP address of the device I was casting to and then went to the apps section in Portmaster, selected my browser, in the rules section I pasted the IP Address into allow Outgoing & Incoming rules. This seemed to work for me. 

> Open the google home app.
> 
> Tap "Devices" at the top of the screen
> 
> Locate the desired Chromecast
> 
> Tap the ‘3 dot’ icon/menu button for your device — It's located at the right of the device
> 
> Select Device Settings
> 
> The IP address and MAC address are located at the bottom of this screen.
> [https://www.reddit.com/r/Chromecast/comments/8nu0d7/how_to_find_chromecasts_ip/dzyenff?utm_source=share&utm_medium=web2x&context=3](https://www.reddit.com/r/Chromecast/comments/8nu0d7/how_to_find_chromecasts_ip/dzyenff?utm_source=share&utm_medium=web2x&context=3)

**Comment by dhaavi** [comment]
> I pasted the IP Address into allow Outgoing & Incoming rules

Yup. That'll work too. Just pasting into the Incoming rules should be enough though.

**Comment by deroverda** [comment]
Sorry for the late reply. I managed to get it to work by enabling a couple of incoming ip-adresses for chrome browser app. Feel free to close. Cheers.

**Comment by dhaavi** [comment]
Thanks for reporting back - great to hear it works!
Closing.
